### DISCUSSION TITLE
Add copy to clipboard ability for full contract ID (not 8 char truncation)

### DISCUSSION THREAD

**Discussion by weslowsk** [siacentral/host-manager#100] [Feature Requests]
In the contract view, make the values in the ID column have a context menu (i.e. by right clicking it) and provide the ability to "copy to clipboard".
I currently have to use the link to SiaStats for each row just to grab the full contract ID. I would rather just copy it from the contract view of Sia Host Manager.

**Comment by n8mgr** [comment]
I run into the exact same issue. Will add this immediately.

**Comment by n8mgr** [comment]
I didn't add copy to clipboard, but you can now select the full contract ID in v1.0.23

**Comment by weslowsk** [comment]
I still see only the 8 character contract ID in v1.0.23, not the full contract ID.

**Comment by n8mgr** [comment]
You're right, I'm not sure how the change didn't make it in..

**Comment by n8mgr** [comment]
Fixed by 3d435c2a130f162c57ad3ea4415c4f93fd493cf3, will be included in v1.0.24.
